draft-moran-suit-mti-algorithms.cddl

```diff
@@ -0,0 +1,127 @@
+SUIT_COSE_tool_tweak /= suit-sha256-hmac-a128-ccm
+SUIT_COSE_tool_tweak /= suit-sha256-es256-hpke-a128gcm
+SUIT_COSE_tool_tweak /= suit-sha256-eddsa-hpke-a128gcm
+SUIT_COSE_tool_tweak /= suit-sha256-hsslms-hpke-a128gcm
+
+SUIT_COSE_tool_tweak /= SUIT_COSE_Profile_HMAC_A128CCM
+SUIT_COSE_tool_tweak /= SUIT_COSE_Profile_ES256_HPKE_A128GCM
+SUIT_COSE_tool_tweak /= SUIT_COSE_Profile_EDDSA_HPKE_A128GCM
+SUIT_COSE_tool_tweak /= SUIT_COSE_Profile_HSSLMS_HPKE_A128GCM
+
+suit-sha256-hmac-a128-ccm = [5, 12]
+suit-sha256-es256-hpke-a128gcm = [-7, 1]
+suit-sha256-eddsa-hpke-a128gcm = [-8, 1]
+suit-sha256-hsslms-hpke-a128gcm = [-46, 1]
+
+SUIT_COSE_Profile_HMAC_A128CCM = SUIT_COSE_Profile<5,12> .and COSE_Messages
+SUIT_COSE_Profile_ES256_HPKE_A128GCM = SUIT_COSE_Profile<-7,1> .and COSE_Messages
+SUIT_COSE_Profile_EDDSA_HPKE_A128GCM = SUIT_COSE_Profile<-8,1> .and COSE_Messages
+SUIT_COSE_Profile_HSSLMS_HPKE_A128GCM = SUIT_COSE_Profile<-46,1> .and COSE_Messages
```

Comment:
Why can't we have a single integer value per profile, so it can be encoded in negotiation like a single COSE Algorithm would be?

Reply:
I can't figure out what a literal encoding would be as specified, can you provide an example?